src/Microsoft.OData.Client/MaterializeFromAtom.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.OData.Client
     #endregion Namespaces
 
     /// <summary>
-    /// Use this class to materialize objects from an application/atom+xml stream.
+    /// Use this class to materialize objects from a stream.
     /// </summary>
     internal class MaterializeAtom : IDisposable, IEnumerable, IEnumerator

```

Comment:
Maybe change "MaterializeAtom" to "MaterializeObject" given the summary doc and also because "MaterializeEntry" represents something different